azure-pipelines.yml

```diff
@@ -53,7 +53,8 @@ steps:
   displayName: 'OSX/Windows CMake'
   condition: or( eq( variables['Agent.OS'], 'Windows_NT'), eq( variables['Agent.OS'], 'Darwin'))
 - script: |
-    cmake --build build --config Release 
+    cmake --build build --config Release
+    cmake --build build --config Release --target style
```

Comment:
Similarly here, what is the intent in adding these?

Perhaps we should add a CI job that checks the formatting via `make check` and fail if the code's not formatted

Reply:
Or, is this just meant to catch a situation where `make style` crashes ?

Reply:
These actually don't do anything due to BLT not finding any formatters... made an issue ( #546) and will address this later.